src/store-api/src/region_request.rs

```diff
@@ -661,23 +667,126 @@ impl From<v1::ChangeColumnType> for ChangeColumnType {
 #[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
 pub enum ChangeOption {
     TTL(Duration),
+    TwscMaxActiveWindowRuns(Option<usize>),
```

Comment:
What about changing these variants into one 

```Rust
Twcs(String)
```

The `String` value is the option name such as `compaction.twcs.max_active_window_runs` etc.

I think it looks much better and we can add new options in the future without adding new variant of `ChangeOption`.

Reply:
I think the draw back would be that we cannot validate the type of value at storage api layer - i.e user can pass max_active_window_runs='abcde' and this will error out at mito engine code path

Reply:
> I think the draw back would be that we cannot validate the type of value at storage api layer - i.e user can pass max_active_window_runs='abcde' and this will error out at mito engine code path (where we handle the alter options).

We can match the option to verify its validity, as the next comment explains.

Reply:
ah i see great call

Reply:
I took a try at implement this approach i found it quite difficult below is my approach:
1. we also need the value of the change, the enum would look like Twcs(String, String)
2. the second comments suggest that we match and validate values at Mito side when handling the alter request, if there is an error we have to return a Mito error which will cause a lot of drift of logic on alter tables. (i.e. ttl change values are parsed and validated when building the request and it will return a metadata error - InvalidRegionOptionChangeRequest).

If our goal is to reduce the variant of ChangeOption. 
I propose a nested enum approach - to parse and validate the values when we are building the request to stay consistent with ttl changes.
```
pub enum ChangeOption {
    TTL(Duration),
    Twcs(TwcsChangeOptions),
}

pub enum TwcsChangeOptions {
       MaxActiveWindowFiles(Option<Duration>),
       ...
}
```